lib/inventory_refresh/save_collection/base.rb

```diff
@@ -22,12 +24,26 @@ def save_inventory_object_inventory(ems, inventory_collection)
         else
           save_inventory(inventory_collection)
         end
-        logger.debug("Saving collection #{inventory_collection}, for the manager: '#{ems.name}'...Complete")
+        logger.debug("----- END ----- Saving collection #{inventory_collection}, for the manager: '#{ems.name}'...Complete")
         inventory_collection.saved = true
       end
 
       private
 
+      # Returns true and sets collection as saved, if the collection should be skipped.
+      #
+      # @param inventory_collection [InventoryRefresh::InventoryCollection] InventoryCollection object we want to save
+      # @return [Boolean] True if processing of the collection should be skipped
+      def skip?(inventory_collection)
+        if inventory_collection.noop?
+          logger.debug("Skipping #{inventory_collection} because it results to noop.")
```

Comment:
because it is empty maybe?

Reply:
I think I might just write 'no operation', since even empty inventory collection could delete records.